### DISCUSSION TITLE
Adding an image to a form

### DISCUSSION THREAD

**Discussion by cirrion-software** [filamentphp/filament#12563] [Ideas]
I dug around for quite a while to find out how to do this. With trial and error I found you can add an image to a form using the Placeholder component. There were a lot of people asking the same question and some very elaborate work arounds on stack overflow etc. For anyone else needing this, I simply used:

```
Placeholder::make('Image')
   ->content(function ($record): HtmlString {
      return new HtmlString("<img src= '" . $record->URL . "')>");
   })
```
May be worth adding this to the documentation.

**Comment by xalabama** [comment]
You are free to create custom components like this to reuse them in your project. Filament provides you an basic set of form components. 

By the way, your solution is already well documented here ;) 
https://filamentphp.com/docs/3.x/forms/layout/placeholder#rendering-html-inside-the-placeholder

**Reply by cirrion-software** [reply]
Yes I know and use custom components but wanted something very simple and the placeholder fulfils this. The solution is not well documented, ok it mentions using HTMLString but the opening paragraph for the placeholder component states _"Placeholders can be used to render **text-only** "fields" within your forms"_

That opening paragraph with the words **text-only** may immediately make you think an image cannot be rendered within.

It's also apparent from the searches and results on Google that a lot of people are searching for a similar simple solution like above. It would be worth mentioning in the documentation.

**Reply by xalabama** [reply]
Hmm, okay its not directly written inside the documentation that you can also render images within the HtmlString method. I think i get your point. For me it was clear enough, so i never have thinked about that. If so many people having trouble with this, it may be a good thing to add this inside the docs. Another idea is to add those snippets inside a kind of a best practise section? 

However if someone searches again for this he hopefully find your snippet. So thanks for your contribution :)
